code/modules/mob/living/status_procs.dm

```diff
@@ -157,6 +158,22 @@
 	if(val_change)
 		update_animations()
 
+// LEGLESS
+
+/mob/living/proc/IsLegless(updating = 1)
```

Comment:
Is this ever used or needed?

Reply:
I'm not sure if they're needed honestly. I only created this section because I saw the "resting" section use that bit of code